extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KnativeConfig.java

```diff
@@ -295,10 +288,6 @@ public Optional<String> getServiceAccount() {
         return serviceAccount;
     }
 
-    public Optional<String> getHost() {
-        return host;
-    }
```

Comment:
It seems that the only motivation for this property was to implement the interface PlatformConfiguration, because as far as I see this property has no effects on the generated resources.